docs/user/introduction.asciidoc

```diff
@@ -173,14 +170,14 @@ them in bulk operations.
 
 {kib} offers a range of security features for you to control who has access to what.
 The security features are automatically turned on when
-{ref}/get-started-enable-security.html[security is enabled in
+{ref}/security-minimal-setup.html[security is enabled in
 {es}]. For a description of all available configuration options,
```

Comment:
This PR targets 8.1, 8.0, and 7.16.

I think the link here makes sense for 7.16, but we should probably link to the "[Start the Elastic Stack with security enabled](https://kibana_118794.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/configuring-stack-security.html)" page (`configuring-stack-security.html`) for 8.0+. WDYT?

Reply:
@jportner Thanks for the comments.  I'll merge these changes, and then revisit the 8.x docs for the changes you mention.